spec/features/user_creates_habit_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+feature "user creates a habit" do
+  context "authenticated user" do
+    scenario "successfully create a new habit" do
+      user = FactoryGirl.create(:user)
+      sign_in_as user
+
+      click_on 'Add habit'
+      fill_in 'Name', with: 'Brushing teeth'
+      check 'Daily'
+      click_on 'Add Habit'
+
+      expect(page).to have_content('Successfully created new habit.  Good luck!')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.